packages/typespec-autorest-canonical/test/additional-properties.test.ts

```diff
@@ -0,0 +1,80 @@
+import { deepStrictEqual, ok } from "assert";
+import { describe, it } from "vitest";
+import { oapiForModel } from "./test-host.js";
+
+describe("typespec-autorestcanonical: Additional properties", () => {
```

Comment:
as we use vitest now, you should be able to remove the top level describe everywhere as its already showing as scoped per file

Reply:
removed the top describe